skops/hub_utils/tests/test_hf_hub.py

```diff
@@ -416,11 +452,15 @@ def test_inference(
 ):
     # test inference backend for classifier and regressor models. This test can
     # take a lot of time and be flaky.
-    client = HfApi()
+    config_path, file_format = config_json
+    if file_format != "pickle":
+        pytest.skip("Unsupported configuration for inference test.")
```

Comment:
We could be more specific:

```suggestion
        pytest.skip(f"Inference only supports pickle at the moment. Given format: {file_format}")
```